### PR TITLE
Fix access on vim.wo

### PR DESCRIPTION
**Pull request by norcalli**

### CODE REVIEW

**Review by justinmk — APPROVED** [review]
seems like we don't have test coverage for this.

**Comment by blueyed** [comment]
> seems like we don't have test coverage for this.

It's because `src/nvim/lua/vim.lua` is compiled into Neovim itself I assume.

**Comment by justinmk** [comment]
> > seems like we don't have test coverage for this.
> 
> It's because `src/nvim/lua/vim.lua` is compiled into Neovim itself I assume.

I meant that no tests were added/adjusted after this change.

File: test/functional/lua/vim_spec.lua [hunk 1]
```diff
@@ -629,22 +629,27 @@ describe('lua stdlib', function()
...
+    vim.cmd "split"
+    vim.api.nvim_win_set_option(0, "cole", 2)
     ]]
     eq(2, funcs.luaeval "vim.wo.cole")
     exec_lua [[
     vim.wo.conceallevel = 0
-    vim.bo[BUF].modifiable = true
     ]]
     eq(0, funcs.luaeval "vim.wo.cole")
+    eq(0, funcs.luaeval "vim.wo[0].cole")
```
Comment (justinmk) [inline test/functional/lua/vim_spec.lua]: in order to test the change, shouldn't this be:

```suggestion
    eq(0, funcs.luaeval "vim.wo.cole")
```

?

Comment (norcalli) [reply test/functional/lua/vim_spec.lua]: Sure, but I just beefed up the tests overall, not just for this test. Testing [0] as well helps.

**Pull request merged**